packages/block-library/src/button/block.json

```diff
@@ -69,6 +69,11 @@
 		"typography": {
 			"fontSize": true,
 			"__experimentalFontFamily": true,
+			"lineHeight": true,
+			"__experimentalFontStyle": true,
+			"__experimentalFontWeight": true,
+			"__experimentalLetterSpacing": true,
```

Comment:
Letter spacing should not allow negative values. Here is an example how it looks if letter space is a negative value.
<img width="1037" alt="image" src="https://user-images.githubusercontent.com/21127788/161366320-e8606116-8b70-4f2f-93eb-befdfb911bae.png">

Reply:
I think this needs a new issue(if one not exists). Could you check and create one @amustaque97 ?

Reply:
Created an issue https://github.com/WordPress/gutenberg/issues/39993

Reply:
Closed the issue #39993 and this review comment is no longer valid. 🙂